modules/UI/util/UIUtil.js

```diff
@@ -231,10 +254,40 @@ const TOOLTIP_POSITIONS = {
      * @param {String} the identifier of the element to hide
      */
     hideElement(id) {
-        if ($("#"+id).hasClass("show"))
-            $("#"+id).removeClass("show");
+        let element;
+        if (id instanceof HTMLElement) {
+            element = id;
+        } else {
+            element = document.getElementById(id);
+        }
+
+        if (!element) {
+            return;
+        }
+
+        let type = this.getElementDefaultDisplay(element.tagName);
+        let className = SHOW_CLASSES[type];
+
+        if(element.classList.contains(className)) {
+            element.classList.remove(className);
+        }
+
+        element.classList.add('hide');
+    },
+
+    /**
+     * Returns default display style for the tag
+     * @param tag
+     * @returns {*}
+     */
+    getElementDefaultDisplay(tag) {
```

Comment:
I thought we can expose this method to the interface of `UIUtils` but we don't use it in the code outside the module. So, let's keep it private for now.